apps/style/common.scss

```diff
@@ -825,6 +825,58 @@ input[type="radio"] {
   font-size: large;
 }
 
+#puzzleRatingButtons {
+
+  $red: #FF3014;
+  $dark_red: #7F150A;
+  $blue: #0094CA;
+  $dark_blue: #00647F;
+  $gray: #333333;
+  $dark_gray: #4D4D4D;
+  // $light_gray comes from shared colors
```

Comment:
Oh dang. Our `colors.scss` still has lots of underscores, but [style guide](https://github.com/thoughtbot/guides/tree/master/style/sass#formatting) we use says use `-` separators.

Might make sense to use `-` in this new code, & for the light_gray line do something like

`$light-gray: $light_gray; // use light gray from shared colors`

Which has the nice property of code-ify-ing the code-y parts of the comment, if that makes sense :)